__tests__/app.test.js

```diff
@@ -207,6 +245,8 @@ describe("/api/articles/:article_id/comments", () => {
         });
     });
 
+    // Because this is a not found type error, the status code should be 404
+    // vvv(to be refactored)
```

Comment:
Ensure you remove comments like this before submitting code for review, it's not required.